build.gradle

```diff
@@ -96,7 +96,7 @@ task shfmtCheck(type: Exec) {
 }
 
 task shellCheck(type: Exec) {
-    commandLine(["shellcheck", "--exclude=SC1090,SC1091"] + shellScripts.getFiles())
+    commandLine(["shellcheck", "--exclude=SC1090,SC1091,SC2148"] + shellScripts.getFiles())
```

Comment:
I'd rather us fix how we build the `shellScripts` variable to not include this new `etc_system_whitelist` file, rather than add this exception.

Reply:
Ah, like how we do:
```
fileTree("upgrade/upgrade-scripts", {
                        exclude "README.md"
              })
```
That seems cleared. I already kicked off integration, is there a way to abort it?

Reply:
If you change the PR branch, it'll automatically cancel bors. i.e. adding a new commit, modifying an existing commit (e.g. `git commit --ammend`), etc.

You can also comment with "bors r-" and it'll cancel bors.

Reply:
I've cancelled the staging build.
I agree that it's better to fix that, I didn't realize we were providing the file list manually.
We should use `shfmt -f .` instead.

Reply:
cool, thanks!

Reply:
> I've cancelled the staging build.

In the future, please use "bors r-", so that it's more clear exactly what is happening and why.

Reply:
> I agree that it's better to fix that, I didn't realize we were providing the file list manually.
> We should use `shfmt -f .` instead.

@pzakha I'm not sure what you mean by this comment. Why should we be using `shfmt -f `?

Reply:
`shfmt -f .` looks for shell scripts in the current directory recursively.
See https://github.com/delphix/delphix-platform/blob/master/Makefile#L52 as an example.

Reply:
Ok, makes sense. Though, it looks like this was designed a little differently, operating over ansible file objects instead of strings as that makefile does. I think I'm going to merge the changes for this review and ask John (the original author) if he thinks it would be clearer to switch to that style.